source/install/self-managed-changelog.md

```diff
@@ -5,14 +5,156 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+
+### Important Upgrade Notes
+ - v5.38 adds fixes for some of the incorrect mention counts and unreads around threads and channels since the introduction of Collapsed Reply Threads (Beta). This fix is done through a SQL migration, and it may take several minutes to complete for large databases. The ``fixCRTChannelMembershipCounts`` fix takes 1 minute and 20 seconds for a database containing approximately 4 million channel memberships and about 130,000 channels. The ``fixCRTThreadCountsAndUnreads`` fix takes about 3 minutes and 30 seconds for a database containing 56367 threads, 124587 thread memberships, and 220801 channel memberships. These are on MYSQL v5.6.51.
+ - Focalboard v0.8.0 (released with Mattermost v5.38.0) requires Mattermost v5.37 due to the new database connection system.
+
+**IMPORTANT:** If you upgrade from a release earlier than v5.37, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### Granular Data Retention Policies (Enterprise E20)
+ - A ``data_retention`` type job can now be run even if the global policy is disabled. The granular (i.e. team and channel-specific) policies will be executed when the data retention job is run. Please note there is a known issue where deleted posts get displayed in channels without new activity after the retention job is run.  This issue is tracked with [this ticket](https://mattermost.atlassian.net/browse/MM-36574).
+
+#### Enhanced User Onboarding Experience
+ - To help new users get started with Mattermost, new Getting Started steps have been added to the onboarding experience. These steps help users to complete their profile, name their teams, configure desktop notifications and invite others to join their team. Additionally, once the onboarding is complete, users are provided with helpful tips to get started with channels, plugins, and more.
+
+#### Playbooks Updates
+ - ``Incident Collaboration`` was rebranded to ``Playbooks``. Also the channel right-hand sidebar is redesigned, our own playbooks are shared as templates, and more triggers and actions were added.
+
+#### Focalboard Updates
+ - Added created-by property and improved performance with shared database connections. Focalboard 0.8.0 requires Mattermost v5.37+ due to a new database connection system.
+
+### Improvements
+
+#### User Interface (UI)
+ - Markdown formatting is now stripped from push notifications.
+ - Enabled the **Set Status** button if the custom status hasn't changed from currently set status.
+ - Improved default rendering of images inserted via the GIF picker.
+ - Small text changes were added to Direct and Group Message menus: 'Mute channel' and 'Edit Channel Header' now reads 'Mute Conversation' and 'Edit Conversation Header'.
+ - Changed the icon from a globe to a lock for private channels.
```

Comment:
I didn't think this is new?

Reply:
@esethna It came from the release note that was added in this PR https://github.com/mattermost/mattermost-webapp/pull/8286.

Reply:
@amyblais ah that PR is referring to the channel autocomplete, which will show private channels after the server PR is merged: https://github.com/mattermost/mattermost-server/pull/17804

No release note necessary until the feature goes in, and then the release note should say "Private channels are now available in the channel autocomplete list"